<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormAnalyze</class>
 <widget class="QWidget" name="FormAnalyze">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Probing signal</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="boxWaveForm"/>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="boxVoltage">
          <property name="suffix">
           <string>V</string>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="maximum">
           <double>5.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>2.200000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QSpinBox" name="boxFrequency">
          <property name="suffix">
           <string>Hz</string>
          </property>
          <property name="minimum">
           <number>50</number>
          </property>
          <property name="maximum">
           <number>20000</number>
          </property>
          <property name="value">
           <number>222</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSlider" name="sliderVoltage">
          <property name="maximum">
           <number>50</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QSlider" name="sliderFrequency">
          <property name="minimum">
           <number>50</number>
          </property>
          <property name="maximum">
           <number>20000</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>1000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,2,0">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="buttonRun">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>Run</string>
         </property>
         <property name="icon">
          <iconset resource="analog_signature_analyzer.qrc">
           <normaloff>:/icons/analyze.ico</normaloff>:/icons/analyze.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="SquareWidgetHolder" name="widget" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <widget class="SignatureView" name="viewSignature">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>40</y>
          <width>344</width>
          <height>192</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="buttonOpenSignature">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>Open signature</string>
         </property>
         <property name="icon">
          <iconset resource="analog_signature_analyzer.qrc">
           <normaloff>:/icons/open.ico</normaloff>:/icons/open.ico</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="buttonLockSignature">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>Lock signature</string>
         </property>
         <property name="icon">
          <iconset resource="analog_signature_analyzer.qrc">
           <normaloff>:/icons/locked.ico</normaloff>
           <normalon>:/icons/unlocked.ico</normalon>:/icons/locked.ico</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="buttonSave">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
         <property name="icon">
          <iconset resource="analog_signature_analyzer.qrc">
           <normaloff>:/icons/save.ico</normaloff>:/icons/save.ico</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SignatureView</class>
   <extends>QGraphicsView</extends>
   <header location="global">widgets/signatureview.h</header>
  </customwidget>
  <customwidget>
   <class>SquareWidgetHolder</class>
   <extends>QWidget</extends>
   <header location="global">widgets/squarewidgetholder.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="analog_signature_analyzer.qrc"/>
 </resources>
 <connections/>
</ui>
